ambari-server/src/main/resources/common-services/SPARK2/2.0.0/package/scripts/params.py

```diff
@@ -198,4 +201,50 @@
   default_fs = default_fs,
   immutable_paths = get_not_managed_resources(),
   dfs_type = dfs_type
- )
+)
+
+
+# livy related config
+
+# livy is only supported from HDP 2.5
+has_livyserver = False
+
+if stack_version_formatted and check_stack_feature(StackFeature.SPARK_LIVY2, stack_version_formatted):
+  livy2_component_directory = Script.get_component_from_role(SERVER_ROLE_DIRECTORY_MAP, "LIVY2_SERVER")
+  livy2_conf = format("{stack_root}/current/{livy2_component_directory}/conf")
+  livy2_log_dir = config['configurations']['livy2-env']['livy2_log_dir']
+  livy2_pid_dir = status_params.livy2_pid_dir
+  livy2_home = format("{stack_root}/current/{livy2_component_directory}")
+  livy2_user = status_params.livy2_user
+  livy2_group = status_params.livy2_group
+  user_group = status_params.user_group
+  livy2_hdfs_user_dir = format("/user/{livy2_user}")
+  livy2_server_pid_file = status_params.livy2_server_pid_file
+
+  livy2_server_start = format("{livy2_home}/bin/livy2-server start")
+  livy2_server_stop = format("{livy2_home}/bin/livy2-server stop")
+  livy2_logs_dir = format("{livy2_home}/logs")
+
+  livy2_env_sh = config['configurations']['livy2-env']['content']
+  livy2_log4j_properties = config['configurations']['livy2-log4j-properties']['content']
+  livy2_spark_blacklist_properties = config['configurations']['livy2-spark-blacklist']['content']
+
+  livy2_kerberos_keytab =  config['configurations']['livy2-conf']['livy2.server.kerberos.keytab']
+  livy2_kerberos_principal = config['configurations']['livy2-conf']['livy2.server.kerberos.principal']
```

Comment:
so, should we change this to 
livy2_kerberos_keytab =  config['configurations']['livy-conf']['livy.server.kerberos.keytab']
+  livy2_kerberos_principal = config['configurations']['livy-conf']['livy.server.kerberos.principal']

it means that we use the configuration of livy. @zjffdu